homeassistant/components/vicare/manifest.json

```diff
@@ -11,5 +11,5 @@
   "documentation": "https://www.home-assistant.io/integrations/vicare",
   "iot_class": "cloud_polling",
   "loggers": ["PyViCare"],
-  "requirements": ["PyViCare==2.32.0"]
+  "requirements": ["PyViCare-neo==0.2.1"]
```

Comment:
If you're not planning to contribute back changes to upstream you should ask GitHub to disconnect your fork from its upstream. That way it's clear to the community that it's a separate project going forward.

Reply:
Please add a description of the due diligence that has been done to continue working with upstream project before labeling the project as unmaintained.

Reply:
> Please add a description to the PR of the due diligence that has been done

done, I hope that is enough.

Reply:
Thanks!

Reply:
Original owner of the lib just responded and agreed to transfer the repo to a dedicated org where we can share the ownership. 
Hope it's not a big deal to change back once the repo is transferred, release is setup and the bugfixes are cherry-picked.

Reply:
That's fine. You can link to this discussion thread in the future PR that changes back the library.